src/components/CardCover.vue

```diff
@@ -0,0 +1,79 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div class="card-cover-wrap" :class="{ rounded }">
+    <slot classes="card-cover">
+      <div class="card-cover card-cover--light" :style="styles.light" />
+      <div class="card-cover card-cover--dark" :style="styles.dark" />
+    </slot>
+  </div>
+</template>
+
+<script>
+import imageAsset from 'docc-render/mixins/imageAsset';
+import { normalizeAssetUrl } from 'docc-render/utils/assets';
+
+const toCSSUrl = url => (url ? `url('${normalizeAssetUrl(url)}')` : undefined);
```

Comment:
Maybe we could extract this function into an utility? we also use them in other places like
`bgStyle` computed property in `Tutorials/Hero.vue`

Reply:
deff a thing we can do..